package.json

```diff
@@ -1,26 +1,44 @@
 {
   "name": "@ordermentum/eslint-config-ordermentum",
-  "version": "1.1.2",
+  "version": "2.0.0-rc2",
   "main": "index.js",
   "repository": "git@github.com:ordermentum/eslint-config-ordermentum.git",
   "author": "John D'Agostino <john@ordermentum.com>",
   "license": "MIT",
   "dependencies": {
-    "@emotion/eslint-plugin": "^11.2.0",
-    "@typescript-eslint/eslint-plugin": "4.33.0",
-    "@typescript-eslint/parser": "4.33.0",
-    "babel-eslint": "10.1.0",
-    "eslint": "7.32.0",
-    "eslint-config-airbnb": "18.2.1",
-    "eslint-config-prettier": "8.6.0",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-jsx-a11y": "6.4.1",
+  },
+  "devDependencies": {
+    "@emotion/eslint-plugin": "11.10",
+    "@typescript-eslint/eslint-plugin": "^5.48.2",
+    "@typescript-eslint/parser": "^5.48.2",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb": "19.0.4",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsx-a11y": "6.7.1",
     "eslint-plugin-mocha": "9.0.0",
-    "eslint-plugin-prettier": "3.4.1",
-    "eslint-plugin-react": "7.24.0",
+    "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-react-hooks": "^4",
-    "eslint-plugin-unicorn": "34.0.1",
-    "prettier": "2.8.3",
-    "typescript": "3.7"
+    "eslint-plugin-react": "7.32.1",
+    "eslint-plugin-unicorn": "45.0.2",
+    "eslint": "^8.32.0",
+    "typescript": "^4.9.4"
```

Comment:
Think it's a good opportunity to be flexible with version ranges now: https://github.com/npm/node-semver#caret-ranges-123-025-004
Especially with minor and patch updates.